generators/client-2/templates/src/main/webapp/app/account/_account.module.ts

```diff
@@ -18,7 +18,13 @@ import {
     PasswordComponent,
     PasswordResetInitComponent,
     PasswordResetFinishComponent,
-    SettingsComponent
+    SettingsComponent,
```

Comment:
Social related items should be added only when socialLogin is enabled, hence it should be wrapped in an ejs condition everywhere